PIPReader/src/test/resources/application.properties

```diff
@@ -23,12 +23,12 @@ requestType=<Request xmlns="urn:oasis:names:tc:xacml:3.0:core:schema:wd-17" Comb
 
 subjectPip={\
     "id" : "PIPReaderEnv",\
-    "className" : "it.cnr.iit.ucs.pipreader.PIPReader",\
+    "name" : "it.cnr.iit.ucs.pipreader.PIPReader",\
     "attributes" : [ {\
         "CATEGORY" : "urn:oasis:names:tc:xacml:1.0:subject-category:access-subject",\
         "EXPECTED_CATEGORY" : "urn:oasis:names:tc:xacml:1.0:subject-category:access-subject",\
         "ATTRIBUTE_ID" : "urn:oasis:names:tc:xacml:1.0:subject:role",\
-        "FILE_PATH" : "subject.txt",\
+        "FILE_PATH" : "../res/pips/subject.txt",\

```

Comment:
I find this .../ to be complicated now we are declaring implementation specific stuff in the config

Reply:
It's the same we just have a single folder for txt file

Reply:
what do you mean by :  we are declaring implementation specific stuff?
the test config just needs to find the file and it's used only in tests.

In the main resource folder any actual use of this should be configured properly.

Reply:
I was thinking, in the previous deleted line (with the path info), the application new where to look at  to get the resource file. It now needs to know the directory structure. Thats what I mean.

Reply:
yeah, I see but It's still a directory path.
In other configurations we had "pips/subject.txt".
I can see that the ../ could be strange but also duplicating all the files it's not clean.

Another solution is not putting any of those files in resources and having them created/deleted on the fly by the tests in before/after scenario.
Actually one of those is currently handled in this way